dashboard/lib/clients/lti_dynamic_registration_client.rb

```diff
@@ -0,0 +1,18 @@
+class LtiDynamicRegistration
```

Comment:
Nice, minor nitpick is to namespace it like:

```
module Lti
  class DynamicRegistration
```

I should have done this with the LTI advantage client but we hadn't started modularizing our LTI classes yet.

Reply:
Ah got it. Will update this.